ansible/roles/nginx/templates/docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: '3.3'
+services:
+    nginx:
```

Comment:
Why is NGINX by itself on a separate Docker-Compose file?

The current NGINX is only reverse-proxying Uptime-Kuma, so they should be both on the same file that way you don't have to manage multiple Docker Networks.